src/Api/Clips.php

```diff
@@ -68,6 +69,17 @@ public function getTopClips($channel = null, $cursor = null, $game = null, $limi
                 throw new TwitchApiException(sprintf('Only a maximum of 10 games can be queried. %d requested.', $count));
             }
         }
+        
+        if ($language) {
+            if (!is_string($language)) {
+                throw new InvalidTypeException('$language', 'string', gettype($language));
```

Comment:
Shouldn't `'$language'` be in double-quotes in order for the variable to be evalulated versus the string `'$language'` itself being used?